### DISCUSSION TITLE
How do you plot the power output of individual turbines in the farm

### DISCUSSION THREAD

**Discussion by henrysanders3** [NatLabRockies/floris#598] [Q&A] [answered]
Hi all,

I am trying to add to this code to plot the power outputs of each turbine. Belos is the code used, which lines can i add to plot the individual turbine power outputs?

![image](https://user-images.githubusercontent.com/78905038/222395627-bb1d00dd-b270-44ff-b8b7-b8dababdfbcb.png)

**Comment by Bartdoekemeijer** [comment, marked as answer]
Hi @henrysanders3. Take a look at example `06_sweep_wind_conditions.py`, which shows how to calculate and plot the power production of individual turbines.
